app/views/podcast_planner/_form_date_calculator.html.erb

```diff
@@ -0,0 +1,83 @@
+<section>
+  <div class="mb-4">
+    <h2 class="fw-bold">Date Calculation of Draft Batch</h2>
+    <p>
+      Calculate release dates for your drafts by selecting which days and weeks you plan to publish, and then select a date range.
+    </p>
+  </div>
+
+  <div class="mb-4">
+    <h3 class="fw-bold">Day selection</h3>
+    <p>Which day of the week do you plan to publish your episodes? Select all that apply.</p>
+    <div class="col-lg-12">
+      <div class="form-floating">
+        <%= form.select :selected_days, day_options, {}, data: {action: "podcast-planner#preview"}, multiple: true %>
+        <%= form.label :selected_days %>
+      </div>
+    </div>
+  </div>
+
+  <div class="mb-4" data-controller="toggle-field">
+    <h3 class="fw-bold">Week selection</h3>
+    <div class="mb-4">
+      <p>Which weeks of each month do you plan to publish? Select all that apply.</p>
+      <div class="col-lg-12" data-toggle-field-target="field" field="periodic">
+        <div class="form-floating">
+          <%= form.select :period, periodic_weeks_options, {}, data: {action: "podcast-planner#preview"} %>
+          <%= form.label :period %>
+        </div>
+      </div>
+      <div class="col-lg-12 d-none" data-toggle-field-target="field" field="monthly">
+        <div class="form-floating">
+          <%= form.select :monthly_weeks, monthly_weeks_options, {}, data: {action: "podcast-planner#preview"}, multiple: true %>
+          <%= form.label :monthly_weeks %>
+        </div>
+      </div>
+    </div>
+
+    <div class="mb-4">
+      <p>Calculate the weeks by:</p>
+      <div class="col-lg-4">
+        <div class="form-floating">
+          <%= form.toggle_field_select :week_condition, week_condition_options, params, class: "form-select form-select-sm", data: {action: "podcast-planner#preview"} %>
+          <%= form.label :week_condition %>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="mb-4" data-controller="toggle-field">
+    <h3 class="fw-bold">Date Range selection</h3>
+    <div class="row mb-4 g-3">
+      <p>How long should episodes recur? Give a date range with a start date, and choose how to calculate the end date.</p>
+      <div class="col-lg-6">
+        <div class="form-floating">
+          <%= form.date_field :start_date, min: DateTime.tomorrow, max: DateTime.tomorrow + 6.months, value: DateTime.tomorrow, data: {action: "podcast-planner#preview"} %>
+          <%= form.label :start_date %>
+        </div>
+      </div>
+      <div class="col-lg-6" data-toggle-field-target="field" field="episodes">
+        <div class="form-floating">
+          <%= form.number_field :number_of_episodes, {min: 1, max: 100, step: 1, value: 10, data: {action: "podcast-planner#preview"}} %>
```

Comment:
same question here with min/max on episode count. i'm not sure there was a max limit in Publish. but probably don't want people bulk creating 1000s of episodes?

and then the default was 10, which i've recreated here.

Reply:
100 is probably sane.  In Publish, I had to artificially limit things because creating episodes is slow (a bunch of API requests running in series).